.travis.yml

```diff
@@ -48,6 +48,7 @@ jobs:
   # See https://github.com/dart-lang/angular/issues/1114.
   allow_failures:
     - env: PKG="examples/github_issues"
+    - env: PKG="examples/hacker_news_pwa"
```

Comment:
This doesn't seem to affect anything...

Reply:
Yes it does. It allows travis to continue to follow-up stages even if this one job had failures.

Reply:
`Your build matrix was set to allow the failure of job 5044.14 so we continued this build to the next stage.`